main.go

```diff
@@ -0,0 +1,121 @@
+/*
```

Comment:
It seems like it's lost the setup of the localmetrics server (for OSD metrics, rather than operator metrics), can it be added back? 

https://github.com/openshift/rbac-permissions-operator/blob/99c383fcd44d081be1918e79ded9d2ff6269dc95/cmd/manager/main.go#L121

Here is how it was done with the OAO v1 migration:
https://github.com/openshift/ocm-agent-operator/blob/0f2a32b2068ea71814aa9c707e1a8eee905ab652/main.go#L98

Reply:
The localmetrics added